content/blog/2021/04/2021-04-20-tekton-plugin.adoc

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: "Easily reuse Tekton and Jenkins X from Jenkins"
+tags:
+- jenkins-x
+- kubernetes
+- pipeline
+- tekton
+- gitops
```

Comment:
```suggestion
- gitops
- interoperability
```

Might be used by @MarckK for such kind of blogposts and quick search in interoperability SIG